### DISCUSSION TITLE
flinkcdc同步tidb遇到的问题

### DISCUSSION THREAD

**Discussion by juecong** [apache/flink-cdc#1329] [Q&A]
1. flinkcdc监控tidb,  按照官网示例，打印时出现"\200\000\001\000\000\000\002\003\000\345\234\250", 如果用record.getValue().toStringUtf8()  或者  new String(record.toByteArray())  就出现乱码了, 如�      ^_r�      
2.相同代码重复运行多次后，就出现了 at org.tikv.cdc.CDCClient.getMinResolvedTs(CDCClient.java:100)
异常， 切换监控表名后正常不报错， 但是切回原表后，数据可以监控出来，但还是在报相同的错误
![1](https://user-images.githubusercontent.com/5485057/176399561-03601d87-47ef-47f8-9bed-d9a13922f7a3.png)
![2](https://user-images.githubusercontent.com/5485057/176399594-89662f3d-35cd-419e-8cf2-fbed410bbe03.png)

**Comment by MengXiangDing** [comment]
这个应该是 tikv-client-java 的bug，tidb 社区也讨论过这个问题，最后也没有声音了。
https://asktug.com/t/topic/903390/3

**Reply by juecong** [reply]
嗯，应该是，

**Reply by tuoluzhe8521** [reply]
解决了吗？我也遇到了相同的问题
